lib/trino-filesystem-azure/src/main/java/io/trino/filesystem/azure/AzureLocation.java

```diff
@@ -32,7 +32,16 @@ class AzureLocation
     private final Location location;
     private final String scheme;
     private final String account;
+    private final String endpoint;
 
+    /**
+     * Creates a new location based on the endpoint, storage account, container and blob path parsed from the location.
+     * <p>
+     * Locations follow the conventions used by
+     * <a href="https://docs.microsoft.com/en-us/azure/storage/blobs/data-lake-storage-introduction-abfs-uri">ABFS URI</a>
+     * that follows the following convention
+     * <pre>{@code abfs://<container-name>@<storage-account-name>.dfs.<endpoint>/<blob_path>}</pre>
```

Comment:
```suggestion
     * Creates a new location based on the endpoint, storage account, container and blob path parsed from the location.
     * <p>
     * Locations follow the conventions used by
     * <a href="https://docs.microsoft.com/en-us/azure/storage/blobs/data-lake-storage-introduction-abfs-uri">ABFS URI</a> syntax:
     * <pre>{@code abfs://<container-name>@<storage-account-name>.dfs.<endpoint>/<blob_path>}</pre>
```

Reply:
`Locations follow the conventions ... that follows the following convention`

Too many follow